packages/react-components/react-card/src/components/Card/useCard.ts

```diff
@@ -28,13 +28,26 @@ export const useCard_unstable = (props: CardProps, ref: React.Ref<HTMLElement>):
 
   const focusAttrs = focusMode !== 'off' ? { tabIndex: 0, ...groupperAttrs } : null;
 
+  const isInteractive = Boolean(
+    ['a', 'button'].includes(as) ||
```

Comment:
`'a'` could be interactive with `href` also

Reply:
But in this case, we are considering all `a` tags, regardless of the href.